README.md

```diff
@@ -458,7 +460,7 @@ Usage:
   golangci-lint run [flags]
 
 Flags:
-      --out-format string           Format of output: colored-line-number|line-number|json|tab|checkstyle|code-climate (default "colored-line-number")
+      --out-format string           Format of output: colored-line-number|line-number|json|tab|checkstyle|code-climate|junit-xml (default "colored-line-number")
```

Comment:
I would include this change in another PR

Reply:
FIxed.